Client/Frontend/Browser/ErrorPageHelper.swift

```diff
@@ -6,16 +6,40 @@ import Foundation
 import WebKit
 import GCDWebServers
 import Shared
+import Storage
 
 class ErrorPageHelper {
     static let MozDomain = "mozilla"
     static let MozErrorDownloadsNotEnabled = 100
 
+    private static let MessageOpenInSafari = "openInSafari"
```

Comment:
It's used in the `if errDomain == ErrorPageHelper.MozDomain` case -- I just factored out the string to a variable here (since we refer to it later in `didReceiveScriptMessage`). A bit less error prone if we don't have to retype these strings in multiple places.